src/test/java/com/mtatgc/bookshelfms/controller/BookControllerTests.java

```diff
@@ -165,10 +166,12 @@ void update_existing_book() throws Exception {
         // assert
         resp.andExpect(status().isOk())
             .andExpect(content().contentType("application/json"))
-            .andExpect(jsonPath("$.id").value(originalBook.getId()))
-            .andExpect(jsonPath("$.title").value(updates.getTitle()))
-            .andExpect(jsonPath("$.genre").value(updates.getGenre()))
-            .andExpect(jsonPath("$.author_id").value(originalBook.getAuthorId()));
+            .andExpect(jsonPath("$.id").value(1))
+            .andExpect(jsonPath("$.title").value("New Name")) // Title should've been updated!
+            .andExpect(jsonPath("$.genre").value("Mystery")) // Genre should've been updated!
+            .andExpect(jsonPath("$.author_id").value(32))
+            .andExpect(jsonPath("$.published_date").value("2024-01-13"));
```

Comment:
I wanted this assertion to be more explicit because I want the format to be specifically hardcoded and tested to prevent future regression.

Reply:
This is a great detail to catch!